orb.yml

```diff
@@ -62,14 +62,16 @@ commands:
           name: Check services are up
           command: |
             cd $E2E_WORKING_DIRECTORY/behave
-            docker-compose build
+            docker-compose build start_applications
```

Comment:
You have `start_applicaiton` twice here. The command below is also calling it. I don't think you need the extra command here. So you can revert it.